### PR TITLE
Add meeting_id filter to some filters 

### PR DESCRIPTION
**Pull request by reiterl**
I am not sure if I understand the issue https://github.com/OpenSlides/openslides-backend/issues/1203 .So I did some update
of filters as example. Before I continue here @jsangmeister  need to have a look.

### CODE REVIEW

**Comment by reiterl** [comment]
Checked min/max datastore calls. They all use meeting_id.

**Review by jsangmeister — APPROVED** [review]
Looks good! Did not check whether any more places exist, but we will see if more come up. Please make sure to also add the meeting id to all future filter-based requests where possible.

**Comment by reiterl** [comment]
I have gone thru filter, min, max, count and exists.
At one exists I was not sure, how to change it: `action/actions/motion/update.py` Perhaps s.o. other looks at it.

**Comment by jsangmeister** [comment]
> I have gone thru filter, min, max, count and exists. At one exists I was not sure, how to change it: `action/actions/motion/update.py` Perhaps s.o. other looks at it.

I assume you mean this:
https://github.com/OpenSlides/openslides-backend/blob/d4818faffbdec4a7e1b0d8f9eb7693ec1209faca/openslides_backend/action/actions/motion/update.py#L115

That should not be an `exist` query, but rather a simple `get_many` on all possible ids to determine if they exist or not. You can go ahead and change this in this PR.

**Comment by reiterl** [comment]
Update the `motion.update` action: replace `exist` call with get_many.

File: openslides_backend/action/actions/motion/update.py [hunk 1]
```diff
@@ -108,17 +108,18 @@ def set_recommendation_extension_reference_ids(
...
-            exists = self.datastore.exists(
-                collection=Collection(collection),
-                filter=FilterOperator("id", "=", int(id_)),
-            )
-            if exists:
+            motion_ids.append(int(id_))
+        gm_request = GetManyRequest(Collection("motion"), motion_ids, ["id"])
+        gm_result = self.datastore.get_many([gm_request]).get(Collection("motion"), {})
+        for motion_id in motion_ids:
+            if motion_id in gm_result:
```
Comment (jsangmeister) [inline openslides_backend/action/actions/motion/update.py:120]: Iterate over the `gm_result` directly to save time

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Minor improvement

File: openslides_backend/action/actions/motion/update.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion/update.py]: Done.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**